packages/deployments/contracts/abi/contracts/core/connext/facets/AssetFacet.sol/AssetFacet.json

```diff
@@ -1,6 +1,6 @@
 [
   "error AssetFacet__addAssetId_alreadyAdded()",
-  "error AssetFacet__addAssetId_noWrapperExists()",
+  "error AssetFacet__addAssetId_nativeAsset()",
```

Comment:
AssetFacet__setWrapper_invalidWrapper no longer used (line 6)

Reply:
These files will all be regenerated (and are autogenerated)